test/commands/create_test.dart

```diff
@@ -107,21 +107,6 @@ void main() {
     expect(projectDir.childFile('tizen/tizen-manifest.xml'), exists);
   }, overrides: <Type, Generator>{});
 
```

Comment:
How about add create test?
```dart
testUsingContext('Can create a C# multi app project', () async {
    final TizenCreateCommand command = TizenCreateCommand();
    final CommandRunner<void> runner = createTestCommandRunner(command);
    await runner.run(<String>[
      'create',
      '--no-pub',
      '--platforms=tizen',
      '--app-type=multi',
      '--tizen-language=csharp',
      projectDir.path,
    ]);

    final File mainDart = projectDir.childFile('lib/main.dart');
    expect(mainDart.readAsStringSync(), contains('Tizen Multi App Demo'));

    final String rawPubspec =
        projectDir.childFile('pubspec.yaml').readAsStringSync();
    final Pubspec pubspec = Pubspec.parse(rawPubspec);
    expect(pubspec.dependencies, contains('tizen_app_control'));
    expect(pubspec.dependencies, contains('messageport_tizen'));

    expect(projectDir.childDirectory('tizen/ui').listSync(), isNotEmpty);
    expect(projectDir.childDirectory('tizen/service').listSync(), isNotEmpty);

    final ProcessResult result = await Process.run(
      'git',
      <String>['diff', '--name-only'],
      workingDirectory: Cache.flutterRoot,
    );
    expect(result.exitCode, 0);
    expect(result.stdout, isEmpty);
  }, overrides: <Type, Generator>{});
```

Reply:
I thought that would be a redundant test case. Compared to the C++ multi app creation test, the only different part is `--tizen-language=csharp`, which doesn't affect the code coverage. If we're to add a test, it would be better to add a test case to `package_test.dart` instead, but it's too much work for me.